app/models/framework/definition/ast/field/options.rb

```diff
@@ -39,7 +39,7 @@ def no_presence_required?
             # YesNo fields don't require an accompanying +presence: true+
```

Comment:
Does this mean this comment is now out of date?

Reply:
It does.